packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -59,7 +71,7 @@ export default class MainProcess {
   private _initTshd() {
     const { binaryPath, flags, homeDir } = this.settings.tshd;
     this.tshdProcess = spawn(binaryPath, flags, {
-      stdio: [null, 'pipe', 'pipe'],
```

Comment:
I wonder why it was set to `null` in the first place. [The docs](https://nodejs.org/api/child_process.html#optionsstdio) say that providing `null` or `undefined` has this effect:

> Use default value. For stdio fds 0, 1, and 2 (in other words, stdin, stdout, and stderr) a pipe is created. For fd 3 and up, the default is `'ignore'`.

Reply:
I don't know tbh, this doesn't seem to have any effect on the collected logs (as you noticed below).